azure-pipelines.yml

```diff
@@ -65,6 +65,32 @@ jobs:
             displayName: "npm build and test"
           - script: npm run test:nolint
             displayName: "npm build and test (no linting)"
+    - job: TestExportedTypes

```

Comment:
I think we could safely rename this to `SmokeTest` as it seems to test not just types, but any error that would show up when setting up a DF app.

Reply:
Yes! I knew `TestExportedTypes` wasn't the best name but I couldn't figure out what to name it. `SmokeTest` is what I was looking for!